src/components/manage/Sharing/__snapshots__/Sharing.test.jsx.snap

```diff
@@ -80,9 +80,7 @@ exports[`Sharing renders a sharing component 1`] = `
             </th>
             <th
               className=""
-            >
-              Contributer
-            </th>
+            />
```

Comment:
@nileshgulia1 it seems there are a few unrelated changes in the PR (tests, CSS, SVG, Icons). Or am I missing something here?

Reply:
@tisto  Yes there are some updated snapshots, and few svg's for default semantic theme, which added when working on `less v3.9.0`  and `semantic-ui-less` upgrades. Although some SVG's and icons from the default semantic theme can be removed.